src/libs/actions/User.ts

```diff
@@ -18,18 +16,24 @@ import * as Session from './Session';
 import * as PersonalDetails from './PersonalDetails';
 import * as OnyxUpdates from './OnyxUpdates';
 import redirectToSignIn from './SignInRedirect';
+import type Login from '../../types/onyx/Login';
+import type OnyxPersonalDetails from '../../types/onyx/PersonalDetails';
+import type {OnyxUpdatesFromServer} from '../../types/onyx';
 
-let currentUserAccountID = '';
+type CustomStatus = {text: string; emojiCode: string; clearAfter?: string};
+type BlockedFromConciergeNVP = {expiresAt: number};
+
+let currentUserAccountID = -1;
 let currentEmail = '';
 Onyx.connect({
     key: ONYXKEYS.SESSION,
     callback: (val) => {
-        currentUserAccountID = lodashGet(val, 'accountID', -1);
-        currentEmail = lodashGet(val, 'email', '');
+        currentUserAccountID = val?.accountID ?? -1;
+        currentEmail = val?.email ?? '';
```

Comment:
Let's rename across the file.
```suggestion
        currentUserAccountID = value?.accountID ?? -1;
        currentEmail = value?.email ?? '';
```